src/test/harness.js

```diff
@@ -554,7 +554,7 @@ async function toggleExceptionLogging() {
 }
 
 async function toggleMappedSources() {
-  return clickElement(".mapped-source");
+  return clickElement(".mapped-source button");
```

Comment:
gosh - it is so easy to break these tests and miss it...

Reply:
Don't worry, we're going to be at a place soon where test failures will be meaningful, I have hope 😄